errors.go

```diff
@@ -0,0 +1,25 @@
+package pkcs12
+
+import "errors"
+
+var (
+	// ErrDecryption represents a failure to decrypt the input.
+	ErrDecryption = errors.New("pkcs12: decryption error")
+
+	// ErrIncorrectPassword is returned when an incorrect password is detected.
+	// Usually, P12/PFX data is signed to be able to verify the password.
+	ErrIncorrectPassword = errors.New("pkcs12: decryption password incorrect")
+)
+
+// NotImplementedError indicates that the input is not currently supported.
+type NotImplementedError struct {
```

Comment:
If user can reach more data held within the error (through exported fields/methods) makes sense to keep it a struct. Otherwise not really needed and in fact making it struct in the future wouldn't be a breaking change.